rskj-core/src/main/java/co/rsk/scoring/InetAddressBlock.java

```diff
@@ -11,30 +11,39 @@
  * <p>
  * Created by ajlopez on 11/07/2017.
  */
+// TODO when InetAddressBlock==subnet gets clarified, update this code-base accordingly, probably some fields and methods will no longer be needed and other will need updates (equals, hashcode...)
 public class InetAddressBlock {

```

Comment:
do you think we could create another class for cidr blocks?.. Just not to mix stuff in one single class. And mark this one as deprecated with a `todo` to refactor / substitute it with the new one in future?

Reply:
Actually, I was in doubts between the two approaches.

Finally I decided to keep it this way as it meant less changes (an easier to review PR) and enabling this functionality in the future for the channel initialisation was as simple as renaming the new method to `contains` and removing the old one.

But I agree we are mixing stuff here and can be confusing if we do not address the TODO in the short term, so I will better split it in two as suggested 👍

Reply:
Done 👍